### PR TITLE
Adding BACK TO TOP links

### PR DESCRIPTION
**Pull request by votemike**
https://jira.dev.bbc.co.uk/browse/PROGRAMMES-5702

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/Ds2013/Page/Schedules/ByDayPage/schedules_by_day_page.html.twig [hunk 1]
```diff
@@ -48,8 +47,15 @@
 {% if schedules_by_day_page.hasBroadcasts %}
     <ol class="list-unstyled g-c-l">
         {% set onairshown = false %}
+        {% set showBackToTop = false %}
         {% for period_name, broadcast_in_period in schedules_by_day_page.BroadcastsGroupedByPeriodOfDay %}
             <li id="{{ period_name }}" class="schedule-skip-link">
+                {% if showBackToTop %}
+                    <a class="pull--right micro link-complex"
+                       href="#top">{{ gelicon('basics', 'to-top', 'centi', 'leading') }}<span
```
Comment (BPScott) [inline src/Ds2013/Page/Schedules/ByDayPage/schedules_by_day_page.html.twig]: Where's the id="top" element that you're linking to?

Comment (votemike) [inline src/Ds2013/Page/Schedules/ByDayPage/schedules_by_day_page.html.twig]: Seems to magically be there, possibly injected by the branding tool? `<span class="radionav-drawer__title top b-g-m">Stations</span>`

Comment (votemike) [inline src/Ds2013/Page/Schedules/ByDayPage/schedules_by_day_page.html.twig]: Actually, that's a class isn't it. In which case it looks like the LIVE site doesn't have an element with the id top either.  
How would you like to proceed?

Comment (votemike) [inline src/Ds2013/Page/Schedules/ByDayPage/schedules_by_day_page.html.twig]: Change it to just `#`?

Comment (BPScott) [inline src/Ds2013/Page/Schedules/ByDayPage/schedules_by_day_page.html.twig]: See if there's a container div somewhere that is suitable for having that ID

**Pull request merged**